README.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Simplex Docs
+ignoreForWeb: true
+---
```

Comment:
we should special-case it

Reply:
... not good to have front matter in the main file

Reply:
Ok I'm removing it now